### PR TITLE
Enhanced Appointment Command 

### PR DESCRIPTION
**Pull request by Py0000**
Add time data field and venue data field.
These 2 are optional fields that can be left out in the user input.

Date field is compulsory.
The order of the 3 fields can be interchanged as long as date field is present.

Delete appointment
`appointment [INDEX] a/delete`
Eg. `appointment 1 a/delete`

Add appointment with date only
`appointment [INDEX] a/[DATE]`
Eg. `appointment 1 a/2021-11-13`

Add appointment with date and time 
`appointment [INDEX] a/[DATE] t/[TIME]`
Eg. `appointment 1 a/2021-11-13 t/22:15`

Add appointment with date and venue
`appointment [INDEX] a/[DATE] v/[VENUE]`
Eg. `appointment 1 a/2021-11-13 v/UTOWN` 

Add appointment with all 3 data fields
`appointment [INDEX] a/[DATE] t/[TIME] v/[VENUE]` 
Eg. `appointment 1 a/2021-11-13 t/20:00 v/SOC COMP-2` 

Update appointment same format as add appointment.
(Have to re-type all 3 fields even if it is not changed).

### CODE REVIEW

File: src/main/java/seedu/fast/logic/commands/AppointmentCommand.java [hunk 1]
```diff
@@ -23,18 +27,36 @@
...
-    public static final String MESSAGE_ADD_APPOINTMENT_SUCCESS = "Added appointment with %1$s: %2$s";
-    public static final String MESSAGE_UPDATE_APPOINTMENT_SUCCESS = "Updated appointment with %1$s: %2$s";
+            + "Parameters (add): INDEX (must be a positive integer), "
+            + PREFIX_APPOINTMENT + "DATE (must be yyyy-mm-dd), "
+            + PREFIX_APPOINTMENT_TIME + "TIME (must be hh:mm (24-hour format)), "
+            + PREFIX_APPOINTMENT_VENUE + "VENUE (recommended to be less than 50 characters long)" + "\n"
+            + PREFIX_APPOINTMENT + "[DATE], "
+            + PREFIX_APPOINTMENT_TIME + "[TIME], "
+            + PREFIX_APPOINTMENT_VENUE + "[VENUE]" + "\n"
+            + "Note: Appointment Time and Appointment Venue are optional, can left blank.\n"
```
Comment (trash-bin99) [inline src/main/java/seedu/fast/logic/commands/AppointmentCommand.java:37]: Minor english error, should be :
Note: Appointment Time and Appointment Venue are optional, can be excluded.

File: src/main/java/seedu/fast/logic/commands/AppointmentCommand.java [hunk 2]
```diff
@@ -23,18 +27,36 @@
...
-            + PREFIX_APPOINTMENT + "2021-10-25 or "
-            + COMMAND_WORD + " 1 "
-            + PREFIX_APPOINTMENT + APPOINTMENT_DELETE_COMMAND;
-
-    public static final String MESSAGE_ADD_APPOINTMENT_SUCCESS = "Added appointment with %1$s: %2$s";
-    public static final String MESSAGE_UPDATE_APPOINTMENT_SUCCESS = "Updated appointment with %1$s: %2$s";
+            + "Parameters (add): INDEX (must be a positive integer), "
+            + PREFIX_APPOINTMENT + "DATE (must be yyyy-mm-dd), "
+            + PREFIX_APPOINTMENT_TIME + "TIME (must be hh:mm (24-hour format)), "
+            + PREFIX_APPOINTMENT_VENUE + "VENUE (recommended to be less than 50 characters long)" + "\n"
```
Comment (trash-bin99) [inline src/main/java/seedu/fast/logic/commands/AppointmentCommand.java:33]: If auto-truncating the string to be 50 chars, should state explicitly that maximum 50 characters is accepted.
Perhaps (has maximum length of 50 characters) would be more clear to the users.

**Review by trash-bin99 — COMMENTED** [review]
LGTM, can be merged after correcting the minor edits to the message, Nice that you added multiple tests to prevent our coverage from going down.

**Comment by Py0000** [comment]
Updated based on recommendation given. 

Improve UI of FAST.

Preview:
image

**Review by trash-bin99 — APPROVED** [review]

**Pull request merged**